### PR TITLE
No need to fetch more blocks once ec.archive.TipHeight passed ec.ceilingHeight

### PR DESCRIPTION
**Pull request by dustinxie**

### CODE REVIEW

File: committee/committee.go [hunk 1]
```diff
@@ -201,6 +201,9 @@ func (ec *committee) Start(ctx context.Context) (err error) {
 	if err = ec.archive.Start(ctx); err != nil {
 		return err
 	}
+	if ec.latestHeightInArchive() >= ec.ceilingHeight {
+		return nil
+	}
```
Comment (yutongp) [inline committee/committee.go:206]: just put before 201 and `ec.terminated = true`

Comment (yutongp) [reply committee/committee.go:206]: at least add `ec.terminated = true`

Comment (dustinxie) [reply committee/committee.go:206]: latestHeightInArchive() calls ec.archive.TipHeight(), so need archive.Start() first?

ok, will add

File: committee/committee.go [hunk 2]
```diff
@@ -202,6 +202,8 @@ func (ec *committee) Start(ctx context.Context) (err error) {
 		return err
 	}
 	if ec.latestHeightInArchive() >= ec.ceilingHeight {
+		zap.L().Info("stop syncing")
+		ec.terminated = true
```
Comment (yutongp) [inline committee/committee.go:206]: nit terminated -> terminatedCarrier

Comment (dustinxie) [reply committee/committee.go:206]: does not hurt to close(ec.terminate) here
otherwise after Stop(), it becomes a dangling channel

**Pull request merged**